frappe_social/frappe_social/doctype/post/post.py

```diff
@@ -0,0 +1,131 @@
+# Copyright (c) 2018, Frappe Technologies and contributors
+# For license information, please see license.txt
+
+
+import frappe
+import requests
+from bs4 import BeautifulSoup
+from frappe.model.document import Document
+
+
+class Post(Document):
+	def on_update(self):
+		if self.is_globally_pinned:
+			frappe.publish_realtime("global_pin", after_commit=True)
+
+	def after_insert(self):
+		frappe.publish_realtime("new_post", self.owner, after_commit=True)
+
+
+@frappe.whitelist()
+def toggle_like(post_name, user=None):
+	liked_by = frappe.db.get_value("Post", post_name, "liked_by")
+	liked_by = liked_by.split("\n") if liked_by else []
+	user = user or frappe.session.user
+
+	if user in liked_by:
+		liked_by.remove(user)
+	else:
+		liked_by.append(user)
+
+	liked_by = "\n".join(liked_by)
+	frappe.db.set_value("Post", post_name, "liked_by", liked_by)
+	frappe.publish_realtime("update_liked_by" + post_name, liked_by, after_commit=True)
```

Comment:
These `publish_realtime` events don't seem to be working correctly.

In the Symo site, if I like a post or add a comment, I can see the response instantly. But when I did that locally with these changes, I had to manually refresh the page to see the effect.

I'm not sure if it's the function calls in the app, or if Frappe changed how `publish_realtime` works in v14.